### ISSUE TITLE
Base32 in DataMatrix

### ISSUE THREAD

**Issue by Em5eline** [metafloor/bwip-js#195]
Hi, 

I am using this library to generate datamatrix containing a base32 string. However, with some strings I get this error:
```
ReferenceError: typeofa is not defined
    at $nt (bwip-js-min.js:41)
    at Object.$1.encCTX (bwip-js-min.js:41)
    at Object.BWIPP.$0.datamatrix (bwip-js-min.js:41)
    at bwip-js-min.js:41
    at Render (bwip-js-min.js:41)
    at Object.ToCanvas [as toCanvas] (bwip-js-min.js:41)
    at TesterComponent.push.qq4z.TesterComponent.initBarcode (tester.component.ts:48)
    at TesterComponent.push.qq4z.TesterComponent.ngOnInit (tester.component.ts:21)
    at callHook (core.js:3281)
    at callHooks (core.js:3251)
```

**Base32 string examples :** 

"37EWN4TGBST3NQZMRZONCZLYGPFEWEQ4NOAVIMAS5P4EY6TNZSY4PXPR3UTMQMZW6AC4EIGC"

"53JNEVAS3DIVHNMBLEL2WVW33BMCHTSZBAYET7KFMOSMYIMFDUHVZ4T7CNLQKLFGEGFTZJ63"

"66OSEF55HBRM5NT3VCEHJFY3NXJWFNWYTEZTCUDSU7YXK2LOPSZZLJ46JPLNLIPDQDGULKI7"

"532Z5XKIE6XYBS5TJJ44K3MSATIFPOPX44I76XIKDDZF4ETAS4PCKPWMPDIMYGCEDMDSSWNC"

"626HART3XS5DSWMGF3U5BFDRPHMYEJWQWQRMH7CXIPYQW7MVT3WC3KEDWBVW5UGBSHFS3MFT"

"36WCADXUREIIBLRLBHXFAEGTN7CRNS2BCTTMYASMV4JL4YYPTK5BYE4V5RA26PM2PDER5PRK"

"53BWVI2YV2UIOGWTAJK7L2H5V5BV32X6MZHBR3XEPMJ5JKK7633AIUGJGYPEEUI2FEMMQTJU"

"352G7AKCE54OVPD3GO2OFT3PBWF2JDEYYCMMPJY6QCY7B5VJOFCV5UVTCHE3K46DQPDNVETJ"

"54S4AQAN4RU5QKZ5FVWDAJHIT2HPNFSJDB6CFTTLVHXZ3Q52DRRCJ3GLYAS3NZK3Y6UWBU6L"

"36DT2XTHIPXEWIUPPIGUWJ2O4FN5OFVYJWFET2JTZMAA3UO7Y2BGHOJ2KARYCLPIQPTOTRCT"

**Code sample :**

```
initBarcode() {

    var strBase32 = base32.stringify(base64.parse(this.randomString(60)));
    try { 
      const newCanvas = document.createElement("canvas");
      newCanvas.width = 300;
      newCanvas.height = 300;
      bwipjs.toCanvas(newCanvas, {
      bcid: "datamatrix", // Barcode type
      text: strBase32, // Text to encode
      includetext: true, // Show human-readable text
      padding: 15,
      backgroundcolor: "ffffff",
     });
     setTimeout(() => {
      document.getElementById("bwipjs-result").innerHTML = "";
      document.getElementById("bwipjs-result").appendChild(newCanvas);
     });
    }
    catch(e) {
      console.error(e);
      console.log(strBase32);
    }
  
 }

  public randomString(length) {
    var result = '';
    var characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
    var charactersLength = characters.length;
    for ( var i = 0; i < length; i++ ) {
       result += characters.charAt(Math.floor(Math.random() * charactersLength));
    }
    return result;
 } 
```

Cheers, 

Émeline M.

**Comment by metafloor** [comment]
This was due to a bug in minifying the module generated from the BWIPP postscript code.  Fixed in version 2.1.1.

**Comment by Em5eline** [comment]
Oh great, thank your for your quick reply.
